### PR TITLE
SDA-3561 - Allow C2 / extensions to retrieve the native window handle

### PR DESCRIPTION
**Pull request by rwestberg**
## Description

In order to support Citrix Media Optimization, the Citrix SDK requires that we can fetch the handle of the native window when playing video, in order to be able to place it correctly. This requires exposing the Electron function GetNativeWindowHandle to the SDA renderer process.

### CODE REVIEW

File: src/demo/win.html [hunk 1]
```diff
@@ -1,87 +1,134 @@
 <html>
-<head>
-    <meta charset="UTF-8">
+  <head>
```
Comment (rwestberg) [inline src/demo/win.html:2]: The unrelated reformatting of this file is done by the pre-commit hook, not sure if I should try to avoid that..

**Review by sbenmoussati — APPROVED** [review]
👍

**Review by KiranNiranjan — APPROVED** [review]
